### PR TITLE
[spv-out] Request proper capabilities for storage image formats.

### PR DESCRIPTION
**Pull request by jimblandy**

### CODE REVIEW

**Comment by jimblandy** [comment]
I kinda think the tests should cover all the WGSL storage formats. It shouldn't be hard to tweak the harness to make this easy. Let me know if you think it's worth it.

File: src/back/spv/writer.rs [hunk 1]
```diff
@@ -858,6 +861,49 @@ impl Writer {
...
+            | If::R16ui
+            | If::R8ui => self.require_any(
+                "storage image format",
+                &[spirv::Capability::StorageImageExtendedFormats],
+            ),
+            If::R64ui | If::R64i => self.require_any(
+                "64-bit integer storage image format",
+                &[spirv::Capability::Int64ImageEXT],
+            ),
+            _ => Ok(()),
```
Comment (kvark) [inline src/back/spv/writer.rs:903]: there aren't hat many left, may be worth just listing hem here explicitly

**Review by kvark — APPROVED** [review]

**Comment by kvark** [comment]
please feel free to merge when ready
I don't think it's worth trying to write a testing infra for SPIR-V storage formats specifically at this point. Maybe later?

**Comment by kvark** [comment]
Published in 0.6.2

**Pull request merged**